app/components/free-response-form/component.js

```diff
@@ -1,75 +1,71 @@
 import Ember from 'ember';
+import {validator, buildValidations} from 'ember-cp-validations';
 
-const formObject = Ember.Object.create({
-    schema: {
-        type: 'object',
-        properties: {
-            activity: {
-                type: 'string',
-                title: 'What were you doing yesterday at 10am/7pm?',
-                maxLength: 75
-            },
-            location: {
-                type: 'string',
-                title: 'Where were you?',
-                maxLength: 75
-            },
-            peoplePresent: {
-                type: 'string',
-                title: 'Who else was present? (If you were alone, please write "alone").',
-                maxLength: 75
-            }
-        }
-    },
-    options: {
-        form: {
-            buttons: {
-            submit: {
-                title: 'Continue',
-                styles: 'btn btn-primary'
-            }
-        }
-        },
-        fields: {
-            activity: {
-                type: 'textarea',
-                constrainMaxLength: true,
-                showMaxLengthIndicator: true,
-                validator: 'required-field'
-            },
-            location: {
-                type: 'textarea',
-                constrainMaxLength: true,
-                showMaxLengthIndicator: true,
-                validator: 'required-field'
-            },
-            peoplePresent: {
-                type: 'textarea',
-                constrainMaxLength: true,
-                showMaxLengthIndicator: true,
-                validator: 'required-field'
-            }
-        },
-        focus: false
+const MAX_LENGTH = 75;
+
+var generateValidators = function(questions) {
+  var validators = {};
+  var presence = validator('presence', {
+      presence:true,
+      message: 'This field is required'
+  });
+  for (var question in questions) {
+      var isOptional = 'optional' in questions[question] && questions[question]['optional'];
```

Comment:
What about
`isOptional = questions[question].optional;`

`optional` will either be falsey (undefined included) or truthy